src/SimpleCalculator.Web/Views/Shared/_CalculationResults.cshtml

```diff
@@ -0,0 +1,52 @@
+@model List<CalculationEntityModel>
```

Comment:
seems, it's duplicate of src/SimpleCalculator.Web/Views/Home/CalculationResults.cshtml